TestList/UnitTest1.cs

```diff
@@ -0,0 +1,73 @@
+// --------------------------------------------------------------------------------------------
+// Training ~ A training program for interns at Metamation, Batch - July 2023
+// Copyright (c) Metamation India.
+// ------------------------------------------------------------------------
+// Program.cs
+// MYLIST<T> 
+// Implement a custom MyList<T> class using arrays as the underlying data structure.
+// --------------------------------------------------------------------------------------------
+using Training;
+namespace list {
+   [TestClass]
+   public class UnitTest1 {
+      MyList<int> list1 = new ();
+      List<int> list2 = new ();
+      [TestMethod]
+      public void TestAdd () {
+         for (int i = 1; i <= 5; i++) list1.Add (i);
+         for (int i = 1; i <= 5; i++) list2.Add (i);

```

Comment:
Why are the loops repeated? You just have to add the items to both lists right? This is prevalent in all other methods as well. Correct all and submit.